src/client.rs

```diff
@@ -64,7 +64,7 @@
 //!
 //! # Example
 //!
-//! ```rust
+//! ```rust, no_run
```

Comment:
Previously, any panics inside `tokio::run` didn't fail the test. Now it does. So, make this compile only.